resources/tests/org.eclipse.core.tests.resources/src/org/eclipse/core/tests/internal/properties/Bug468PerformanceTest.java

```diff
@@ -158,9 +159,18 @@ public void test() throws CoreException {
 		}, this.project, IWorkspace.AVOID_UPDATE, new NullProgressMonitor());
 
 		long maxTime = TimeUnit.MILLISECONDS.convert(1, TimeUnit.MINUTES);
+		long maxTimeMacOS = TimeUnit.MILLISECONDS.convert(75, TimeUnit.SECONDS);
+
 		assertTrue("The expected min time(ms): " + 0 + ", actual time(ms): " + timeTakenForDeletingFiles[0],
 				0 <= timeTakenForDeletingFiles[0]);
-		assertTrue("The expected max time(ms): " + maxTime + ", actual time(ms): " + timeTakenForDeletingFiles[0],
-				timeTakenForDeletingFiles[0] <= maxTime);
+
+		if (OS.isWindows() || OS.isLinux()) {
+			assertTrue("The expected max time(ms): " + maxTime + ", actual time(ms): " + timeTakenForDeletingFiles[0],
+					timeTakenForDeletingFiles[0] <= maxTime);
```

Comment:
I know this code existed like that, but I would actually prefer a change to a matcher-style assertions when touching the code anyway, as it is much easier to read and maintain, like: `assertThat("expected time exceeded", timeTakenForDeletingFiles[0], lessThanOrEqualTo(maxTime))` (using hamcrest MatcherAssert.assertThat and Matchers.lessThanOrEqualTo).